tests/unit/test_bids_utils.py

```diff
@@ -81,3 +86,66 @@ def test_bidsmap_scans_run_plus():
     scans = [("1", "run+"), ("2", "run+"), ("3", "run+")]
 
     assert bidsmap_scans(scans) == [("1", "run-01"), ("2", "run-02"), ("3", "run-03")]
+
+
+def test_bidsify_dicom_headers(mocker):
+    """Test bidsify_dicom_headers without ProtocolName field"""
+    dataset = mocker.MagicMock()
+    dataset.__contains__.return_value = False
```

Comment:
2. Here is where we create our new mock that represents the `dataset` variable in `bidsify_dicom_headers`. The `MagicMock` function creates a new `MagicMock` object, which is just a mock object that has all of the dunder methods. We need to use a `MagicMock` here because we need to mock `__contains__`. The `__contains__` method is used by the statement `"ProtocolName" not in dataset`